### ISSUE TITLE
Taxi dataset will not load

### ISSUE THREAD

**Issue by lostella** [awslabs/gluon-ts#2846]
## To Reproduce

With a fresh dataset folder (e.g. `rm -r ~/.gluonts/datasets/taxi_30min`)

```python
from gluonts.dataset.repository import get_dataset
dataset = get_dataset("taxi_30min")
```

## Error message or code output

```
Traceback (most recent call last):
  File "/Users/stellalo/gluonts/src/gluonts/dataset/jsonl.py", line 127, in __iter__
    yield json.loads(line)
orjson.JSONDecodeError: str is not valid UTF-8: surrogates not allowed: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/stellalo/gluonts/src/gluonts/dataset/repository/datasets.py", line 360, in get_dataset
    dataset_path = materialize_dataset(
  File "/Users/stellalo/gluonts/src/gluonts/dataset/repository/datasets.py", line 307, in materialize_dataset
    dataset_recipe(**kwargs)
  File "/Users/stellalo/gluonts/src/gluonts/dataset/repository/_gp_copula_2019.py", line 131, in generate_gp_copula_dataset
    train_data = get_data(dataset_path / "train", ds_info)
  File "/Users/stellalo/gluonts/src/gluonts/dataset/repository/_gp_copula_2019.py", line 157, in get_data
    return [
  File "/Users/stellalo/gluonts/src/gluonts/dataset/repository/_gp_copula_2019.py", line 157, in <listcomp>
    return [
  File "/Users/stellalo/gluonts/src/gluonts/dataset/__init__.py", line 56, in __iter__
    yield from self.iter_sequential()
  File "/Users/stellalo/gluonts/src/gluonts/dataset/__init__.py", line 47, in iter_sequential
    yield from dataset
  File "/Users/stellalo/gluonts/src/gluonts/dataset/jsonl.py", line 129, in __iter__
    raise GluonTSDataError(
gluonts.exceptions.GluonTSDataError: Could not read json line 0, b'\x00\x05\x16\x07\x00\x02\x00\x00Mac OS X        \x00\x02\x00\x00\x00\t\x00\x00\x002\x00\x00\x00\xa3\x00\x00\x00\x02\x00\x00\x00\xd5\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00ATTR\x00\x00\x00\x00\x00\x00\x00\xd5\x00\x00\x00\x98\x00\x00\x00=\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x00\x98\x00\x00\x00=\x00\x00\x15com.apple.quarantine\x00q/0081;5e2617ec;Firefox;E5ED4C58-99FA-4D71-B3F5-DBCBF1604BAA\x00'
```

Note that I have `orjson` installed, and that is what GluonTS picks up 

## Environment
- Operating system: macos
- Python version: 3.8.13
- GluonTS version: `dev`
- JSON libraries installed: `orjson`

**Comment by jaheba** [comment]
This looks weird:

```
'\x00\x05\x16\x07\x00\x02\x00\x00Mac OS X 
```

**Comment by jaheba** [comment]
This is my local folder structure when executing above code:

```
taxi_30min
├── ._metadata
├── ._test
├── ._train
├── metadata
│  ├── ._metadata.json
│  └── metadata.json
├── test
│  ├── ._test.json
│  └── test.json
└── train
   ├── ._train.json
   └── train.json
```

Maybe it's an issue with safe_extractall?

**Comment by abdulfatir** [comment]
Interesting that another dataset has this problem. I fixed this for Wiki2k but this looks like the same problem: #2805

**Comment by abdulfatir** [comment]
Ref: https://stackoverflow.com/questions/8766730/tar-command-in-mac-os-x-adding-hidden-files-why

TL;DR: `export COPYFILE_DISABLE=true` before creating the tarball on MacOS.

**Comment by jaheba** [comment]
I guess we can handle this client side by ignoring hidden files in our dataset loading by default?

**Comment by abdulfatir** [comment]
That's probably the better way.

**Comment by lostella** [comment]
Any takers?

**Comment by jaheba** [comment]
#2847
